File: test/testMerger.py
from section5 import *

var1 = ['Bat', 'Wonder', 'Iron', 'Bruce', 'Super']; 
var2 = ['man', 'Woman', 'Man', 'Banner', 'Man'];

def test_mergeHandler():
    return mergeHandler(var1, var2, merge)
print (test_mergeHandler())


def test_merge2Single():
    return merge2Single(var1, var2)
print (test_merge2Single())

'''
def test_merge2Ogject():
     return merger2Object(var1, var2)
print (test_merge2Object())
'''

File: test/testMerger.js
const merger = require('./merge');

var1 = ['Bat', 'Wonder', 'Iron', 'Bruce', 'Super']; 
var2 = ['man', 'Woman', 'Man', 'Banner', 'Man'];

function test_mergeHandler(){
    return merger.mergeHandler(var1, var2, merger.merge);
}
console.log(test_mergeHandler());

function test_merge2Single(){
    return merger.merge2Single(var1, var2);
}
console.log(test_merge2Single());

function test_merge2Object(){
    return merger.merge2Object(var1, var2);
}
console.log(test_merge2Object());

// const testFuncMerger = testMerge();
// function testMerge() {
//     return [
//      test_mergeHandler,
//      test_merge2Single,
//      test_merge2Object
//     ].forEach(func => {
//         func.call();
//     });
// }

